src/db/DBConnection.ts

```diff
@@ -15,13 +14,14 @@ const connectionOptions: ConnectionOptions = {
     ssl: isProduction,
   },
   host: process.env.DB_HOST,
-  port: isProduction ? parseInt(process.env.DB_PORT) : 5432,
+  port: isProduction ? +process.env.DB_PORT : 5432,
```

Comment:
I used parseInt instead of "+" for readability but if "+" is preferred then its all good

Reply:
The `parseInt()` function was prohibited in the style guidelines, I think this should wokr the same